Atomic/atomic.py

```diff
@@ -678,6 +679,24 @@ def get_vulnerable_ids(self):
         except IOError:
             return []
 
+    def get_local_tokens(self):
+        if len(self.local_tokens) < 1:
+            self.local_tokens = self.load_local_tokens()
+        return self.local_tokens
+
+    @staticmethod
+    def load_local_tokens():
+        tokens = {}
+        token_file_name = os.path.expanduser('~/.docker/config.json')
+        if not os.path.exists(token_file_name):
+            return {}
+        with open(token_file_name) as token_file:
+            token_data = json.load(token_file)
+        for registry in token_data['auths']:
+            tokens[registry] = token_data['auths'][registry]['auth']
```

Comment:
Should this be in a try block?  Potential errors in the auth.  If the auths and auth do not exists should we return {}